### PR TITLE
Layout page

### PR DESCRIPTION
**Pull request by ryanybrennan**
Still need to add pictures to the carousel on the landing page.
This commit will need to be merged before the other front end commits, since it contains the file structure and static controller for the front end and to avoid merge conflicts.

### CODE REVIEW

File: .idea/ad410project.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/ad410project.iml]: please remove this file addition. instead create a .gitignore file that contains `.idea/*`

File: .idea/deployment.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/deployment.xml]: please remove this file addition

File: .idea/inspectionProfiles/Project_Default.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<component name="InspectionProjectProfileManager">
```
Comment (mcontado) [inline .idea/inspectionProfiles/Project_Default.xml]: please remove this file addition

File: .idea/jsLibraryMappings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/jsLibraryMappings.xml]: please remove this file addition

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/modules.xml]: please remove this file addition

File: .idea/php.xml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/php.xml]: please remove this file addition

File: .idea/vcs.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/vcs.xml]: please remove this file addition

File: .idea/webServers.xml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcontado) [inline .idea/webServers.xml]: please remove this file addition

**Review by mcontado — CHANGES_REQUESTED** [review]
Please remove all files that was added under .idea folder. Create a .gitignore file that will ignore these files.

File: views/layout.php [hunk 9]
```diff
@@ -1,14 +1,21 @@
...
-    </header>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <title>Kamps For Kids</title>
+    <link rel="stylesheet" href="views/styles/styles.css">
```
Comment (mcontado) [inline views/layout.php:11]: in your local, is it not showing "cannot resolve directory 'views'" ?

**Review by mcontado — APPROVED** [review]

**Review by mviencek — APPROVED** [review]
great job.  take a look at the sticky footer for bootstrap.  it works great

**Pull request merged**